docker/run.sh

```diff
@@ -10,12 +10,25 @@ cd "$( dirname "${BASH_SOURCE[0]}" )"
 # Setup env variables with VMm's ip
 source ../aws-infrastructure/set-env.sh
 
-# Pull images and tag them on client VM
+# Copy executable script
 scp -i ../aws-infrastructure/aws_ssh_key -oStrictHostKeyChecking=no ./upload.py ubuntu@$PUBLIC_CLIENT_IP_ADDR:/home/ubuntu/upload.py
-ssh -i ../aws-infrastructure/aws_ssh_key -oStrictHostKeyChecking=no ubuntu@$PUBLIC_CLIENT_IP_ADDR <<'ENDSSH'
-set -x
-/home/ubuntu/upload.py
-ENDSSH
+scp -i ../aws-infrastructure/aws_ssh_key -oStrictHostKeyChecking=no ./upload.py ubuntu@$PUBLIC_SERVER_IP_ADDR:/home/ubuntu/upload.py
+
+# Pull images and tag them on client VM
+ssh -i ../aws-infrastructure/aws_ssh_key -oStrictHostKeyChecking=no ubuntu@$PUBLIC_CLIENT_IP_ADDR "source instance-env.sh; /home/ubuntu/upload.py pull"
```

Comment:
@Sammers21 maybe we can just add public key to known hosts to trust ssh connection and remove `-oStrictHostKeyChecking=no` from all `scp` calls?
```bash
echo -n "$PUBLIC_CLIENT_IP_ADDR " >> ~/.ssh/known_hosts
cat ../aws-infrastructure/aws_ssh_key.pub >> ~/.ssh.known_hosts
```

Reply:
During the debug process the script is called a huge amount of times, which means that users `~/.ssh.known_hosts` file will be messed up

Reply:
@Sammers21 you just need to update public key here only once per host, so you can put it before all `ssh` and `scp` calls

Reply:
And after the first connect attempt the host is added.